pkg/controllers/member/serviceexport/controller.go

```diff
@@ -532,6 +532,15 @@ func (r *Reconciler) markServiceExportAsValid(ctx context.Context, svcExport *fl
 	conflictCond := meta.FindStatusCondition(svcExport.Status.Conditions, string(fleetnetv1alpha1.ServiceExportConflict))
 	if condition.EqualCondition(validCond, expectedValidCond) &&
```

Comment:
would condition.EqualConditionWithMessage work here?

Reply:
nope we cannot, even the message is different and conflict Condi is not nil, we don't reset the conflict condition to unknown in this case.